src/libs/ReportUtils.js

```diff
@@ -2141,6 +2142,25 @@ function isReportDataReady() {
     return !_.isEmpty(allReports) && _.some(_.keys(allReports), (key) => allReports[key].reportID);
 }
 
+/**
+ * @param {object} report
+ * @returns {boolean}
+ */
+function isSingleTransactionThreadReport(report) {
+    const parentReportAction = ReportActionsUtils.getParentReportAction(report);
+    if (_.isEmpty(parentReportAction)) {
+        if (report.parentReportActionID) {
+            Log.alert('parentReportActionID found, but no reportAction exists locally for this ID!');
+        } else {
+            Log.info('No parentReportActionID for this report. Not a thread.');
+        }
```

Comment:
idk if we need these logs I just added them for testing because I want to see whether there is a `parentReportActionID` getting returned at all.

I also cannot get this to work locally yet. The report is not showing up with a `parentReportActionID` field.